core/src/Kokkos_View.hpp

```diff
@@ -1577,7 +1579,8 @@ class View : public ViewTraits<DataType, Properties...> {
       const Impl::ViewCtorProp<P...>& arg_prop,
       typename std::enable_if<Impl::ViewCtorProp<P...>::has_pointer,
                               typename traits::array_layout>::type const&
-          arg_layout)
+          arg_layout,
+      int check_input_args = 0)
```

Comment:
```suggestion
      bool check_input_args = false)
```
Why isn't true the default?